clients/include/blas1/testing_rotm_batched.hpp

```diff
@@ -118,9 +122,10 @@ void testing_rotm_batched(const Arguments& arg)
     hipblas_init_vector(hy, arg, hipblas_client_alpha_sets_nan, false);
     hipblas_init_vector(hdata, arg, hipblas_client_alpha_sets_nan, false);
 
-    for(int b = 0; b < batch_count; b++)
+    for(int64_t b = 0; b < batch_count; b++)
     {
-        ref_rotmg<T>(&hdata[b][0], &hdata[b][1], &hdata[b][2], &hdata[b][3], hparam[b]);
+        int b2 = b;
+        ref_rotmg<T>(&hdata[b2][0], &hdata[b2][1], &hdata[b2][2], &hdata[b2][3], hparam[b2]);
```

Comment:
Not sure why you do this int b2, maybe you haven't rebased.  If it is missing you can add another operator[](int64_t) I am pretty sure I added to something just can't recall what.

Reply:
Yeah it's probably been fixed already. I'll change it all back when I enable full testing.